omero/sysadmins/unix/install-web-trial.txt

```diff
@@ -0,0 +1,348 @@
+OMERO.web WSGI deployment (TRIAL)
+=================================
+
+OMERO.web is the web application component of the OMERO platform which
+allows for the management, visualization (in a fully multi-dimensional
+image viewer) and annotation of images from a web browser. It also
+includes the ability to manage users and groups.
+
+Deploying OMERO.web separately to OMERO.server is recommended as they
+perform best under different circumstances and require a different set of
+dependencies. This guide uses the example of deploying with nginx and
+gunicorn.
+
+Prerequisites
+-------------
+
+-  Python 2.7
+
+   -  :pypi:`virtualenv` (optional) tool to create isolated Python environments
+
+   -  :pypi:`PyPI <>` a package management system used to install and manage
+      software packages written in Python. PyPI is already installed if
+      you are using Python 2 >=2.7.9
+
+   -  `omego <https://github.com/ome/omego>`_ OME package management system
+
+   -  `ZeroC`_ IcePy 3.6 (IcePy 3.5 is also supported although requires global
+      installation from RPM)
```

Comment:
> Maybe I'm unusual, but I much prefer to use my distribution's package manager rather than that and an extra package manager for every language I need extra stuff for: pip, cpan, hackage, etc. It also makes it simpler to restore state to a fresh system.

An exception here, @mtbc, is that I think we're quite close to having a requirements.txt shipped with OMERO, so that on every upgrade we would say, "please run `venv/bin/pip install -r requirements.txt` to have the latest" which isn't necessarily something you'd want to do with your wider system.